_includes/0.1/left_sidebar.html

```diff
@@ -29,6 +29,7 @@
         <a href="{% link docs/0.1/grid_schema_family_specification.md %}">Grid Schema</a>
         <a href="{% link docs/0.1/grid_track_and_trace_family_specification.md %}">Grid Track and Trace</a>
         <a href="{% link docs/0.1/pike_transaction_family.md %}">Pike</a>
+        <a href="{% link docs/0.1/grid_on_splinter.md %}">Grid on Splinter</a>
```

Comment:
Please change the link text to "Running Grid on Splinter", to indicate that this is a how-to topic.  (Also, we might need the plain G-on-S title for a new topic under Getting Started.)